### ISSUE TITLE
react-native-is-edge-to-edge  on web

### ISSUE THREAD

**Issue by dppo** [zoontek/react-native-edge-to-edge#33]
### Before submitting a new issue

- [X] I tested using the latest version of the library, as the bug might be already fixed.
- [X] I tested using a [supported version](https://github.com/reactwg/react-native-releases/blob/main/docs/support.md) of react native.
- [X] I checked for possible duplicate issues, with possible answers.

### Bug summary

✘ [ERROR] No matching export in "node_modules/react-native-web/dist/index.js" for import "TurboModuleRegistry"

    node_modules/react-native-is-edge-to-edge/dist/index.mjs:1:19:
      1 │ import { Platform, TurboModuleRegistry } from 'react-native';
        ╵                    ~~~~~~~~~~~~~~~~~~~

### Library version

0.1.0

### Environment info

```shell
react-native-web
```

### Steps to reproduce

react-native-web TurboModuleRegistry

### Reproducible sample code

```js
use react-native-is-edge-to-edge in react-native-web
```

**Comment by zoontek** [comment]
Fixed in [0.1.2](https://github.com/zoontek/react-native-edge-to-edge/releases/tag/0.1.2)

**Comment by dppo** [comment]
> Fixed in [0.1.2](https://github.com/zoontek/react-native-edge-to-edge/releases/tag/0.1.2)

new error in 0.1.2

```
Failed to resolve entry for package "react-native-is-edge-to-edge". The package may have incorrect main/module/exports specified in its package.json. [plugin vite:dep-pre-bundle]

package.json use "module": "dist/index.mjs", but no index.mjs file
```

**Comment by zoontek** [comment]
Oops, forgot to add back `esm` to build format. Fixed now.
